plugins/services/src/js/components/modals/NewCreateServiceModalForm.js

```diff
@@ -422,10 +451,38 @@ class NewCreateServiceModalForm extends Component {
     ];
   }
 
+  /**
+   * This function filters the error list in order to keep only the
+   * errors that should be displayed to the UI.
+   *
+   * @returns {Array} - Returns an array of errors that passed the filter
+   */
+  getUnmutedErrors() {
```

Comment:
I was hoping that we can generalize it (e.g. `filterErrorsByPaths`) and get rid of the `CONSTANTLY_UNMUTED_ERRORS` solution. This definitely not blocking this PR but we should only add methods to a component if the method actually belongs to the component and not if it's only used by the component. This will help us to better separate things.